docs/topics/cli-args.adoc

```diff
@@ -69,9 +69,9 @@ Do not overwrite a report output directory that contains important information.
 |--source |A space-delimited list of one or more source technologies, servers, platforms, or frameworks to migrate from. This argument, in conjunction with the `--target` argument, helps to determine which rulesets are used. Use the `--listSourceTechnologies` argument to list all available sources.
 |--sourceMode |Flag to indicate that the application to be evaluated contains source files rather than compiled binaries.
 |--target |A space-delimited list of one or more target technologies, servers, platforms, or frameworks to migrate to. This argument, in conjunction with the `--source` argument, helps to determine which rulesets are used. Use the `--listTargetTechnologies` argument to list all available targets.
-|--userIgnorePath |Specify a location, in addition to `${user.home}/.mta/ignore/`, for {ProductShortName} to identify files that should be ignored.
-|--userLabelsDirectory |Specify a location for {ProductShortName} to look for custom Target Runtime Labels. The value can be a directory containing label files or a single label file. The Target Runtime Label files must use either the [x-]`.windup.label.xml` or [x-]`.mta.label.xml` suffix. The shipped Target Runtime Labels are defined within `$MTA_HOME/rules/migration-core/core.windup.label.xml`.
-|--userRulesDirectory |Specify a location, in addition to `<MTA_HOME>/rules/` and `${user.home}/.mta/rules/`, for {ProductShortName} to look for custom {ProductShortName} rules. The value can be a directory containing ruleset files or a single ruleset file. The ruleset files must use the [x-]`.windup.xml` or [x-]`.mta.xml` suffix.
+|--userIgnorePath |Specify a location, in addition to `${user.home}/.{LC_PSN}/ignore/`, for {ProductShortName} to identify files that should be ignored.
```

Comment:
Did QE check all these links, rendered as mtr?

Reply:
@RichardHoch 
No

Reply:
Again, then write duplicate the file, add mtr to the file name, write out MTR (or mtr, etc.) and hope for the best. This is why we are publishing the documents before the splash page works -- so we can fix mistakes before GA.